hack/prepare-release-branch.sh

```diff
@@ -0,0 +1,68 @@
+# !/bin/bash
+#
+# Script runs below steps to prepare a new release branch:
+# - Fetch the latest main branch from remote origin (it must point to github.com/intel/trusted-certificate-issuer).
```

Comment:
github.com/intel/trusted-attestation-controller

Reply:
Fixed it now.